Sources/PT.PM.MySqlParseTreeUst/MySqlAntlrConverter.cs

```diff
@@ -0,0 +1,3034 @@
+
+        public Ust VisitAlterUpgradeName([NotNull] MySqlParser.AlterUpgradeNameContext context)
+        {
+            return VisitChildren(context);
+        }
+
+        public Ust VisitAlterUser([NotNull] MySqlParser.AlterUserContext context)
+        {
+            return VisitChildren(context);
+        }
+
+        public Ust VisitAlterUserMysqlV56([NotNull] MySqlParser.AlterUserMysqlV56Context context)
+        {
+            return VisitChildren(context);
+        }
+
+        public Ust VisitAlterUserMysqlV57([NotNull] MySqlParser.AlterUserMysqlV57Context context)
+        {
+            return VisitChildren(context);
+        }
+
+        public Ust VisitAlterView([NotNull] MySqlParser.AlterViewContext context)
+        {
+            return VisitChildren(context);
+        }
+
+        public Ust VisitAnalyzeTable([NotNull] MySqlParser.AnalyzeTableContext context)
+        {
+            return VisitChildren(context);
+        }
+
+        public Ust VisitAssignmentField([NotNull] MySqlParser.AssignmentFieldContext context)
+        {
+            return VisitChildren(context);
+        }
+
+        public Ust VisitAtomTableItem([NotNull] MySqlParser.AtomTableItemContext context)
+        {
+            return VisitChildren(context);
+        }
+
+        public Ust VisitAuthPlugin([NotNull] MySqlParser.AuthPluginContext context)
+        {
+            return VisitChildren(context);
+        }
+
+        public Ust VisitAutoIncrementColumnConstraint([NotNull] MySqlParser.AutoIncrementColumnConstraintContext context)
+        {
+            return VisitChildren(context);
+        }
+
+        public Ust VisitBeginWork([NotNull] MySqlParser.BeginWorkContext context)
+        {
+            return VisitChildren(context);
+        }
+
+        public Ust VisitBetweenPredicate([NotNull] MySqlParser.BetweenPredicateContext context)
+        {
+            return VisitChildren(context);
+        }
+
+        public Ust VisitBinaryComparasionPredicate([NotNull] MySqlParser.BinaryComparasionPredicateContext context)
+        {
+            return VisitChildren(context);
+        }
+
+        public Ust VisitBinaryExpressionAtom([NotNull] MySqlParser.BinaryExpressionAtomContext context)
+        {
+            return VisitChildren(context);
+        }
+
+        public Ust VisitBinlogStatement([NotNull] MySqlParser.BinlogStatementContext context)
+        {
+            return VisitChildren(context);
+        }
+
+        public Ust VisitBitExpressionAtom([NotNull] MySqlParser.BitExpressionAtomContext context)
+        {
+            return VisitChildren(context);
+        }
+
+        public Ust VisitBitOperator([NotNull] MySqlParser.BitOperatorContext context)
+        {
+            return VisitChildren(context);
+        }
+
+        public Ust VisitBlockStatement([NotNull] MySqlParser.BlockStatementContext context)
+        {
+            var statements = new List<Statement>();
+            for (int i = 0; i < context.ChildCount; i++)
```

Comment:
Is it possible to iterate through all children with `foreach` and `children` collection?